styles/web/_vars.scss

```diff
@@ -666,60 +178,34 @@ $sprk-btn-disabled-border-color: transparent !default;
-$sprk-btn-secondary-disabled-border-style: $sprk-btn-disabled-border-style !default;
-/// The border width of the Secondary Button component when disabled.
-$sprk-btn-secondary-disabled-border-width: $sprk-btn-disabled-border-width !default;
 /// The box shadow of the Secondary Button component when disabled.
 $sprk-btn-secondary-disabled-shadow: none !default;
-/// The box shadow of the Tertiary Button component when disabled.
-$sprk-btn-tertiary-disabled-shadow: none !default;
-/// The border value of the Tertiary Button component when disabled.
-$sprk-btn-tertiary-disabled-border-color: $sprk-black-tint-50 !default;
-/// The border style of the Tertiary Button component when disabled.
-$sprk-btn-tertiary-disabled-border-style: $sprk-btn-disabled-border-style !default;
-/// The border width of the Tertiary Button component when disabled.
-$sprk-btn-tertiary-disabled-border-width: $sprk-btn-disabled-border-width !default;
-/// The box shadow of the Quaternary Button component when disabled.
-$sprk-btn-quaternary-disabled-shadow: none !default;
-/// The border value of the Quaternary Button component when disabled.
-$sprk-btn-quaternary-disabled-border-color: $sprk-black-tint-50 !default;
-/// The border style of the Quaternary Button component when disabled.
-$sprk-btn-quaternary-disabled-border-style: $sprk-btn-disabled-border-style !default;
-/// The border width of the Quaternary Button component when disabled.
-$sprk-btn-quaternary-disabled-border-width: $sprk-btn-disabled-border-width !default;
-/// The border of the Button component when disabled on hover.
-$sprk-btn-disabled-hover-border: $sprk-btn-disabled-border-width
-  $sprk-btn-disabled-border-style $sprk-btn-disabled-border-color !default;
 /// The border of the Secondary Button component when disabled on hover.
 $sprk-btn-secondary-disabled-hover-border: $sprk-btn-secondary-disabled-border-width
   $sprk-btn-secondary-disabled-border-style
   $sprk-btn-secondary-disabled-border-color !default;
-/// The border of the Tertiary Button component when disabled on hover.
-$sprk-btn-tertiary-disabled-hover-border: $sprk-btn-tertiary-disabled-border-width
-  $sprk-btn-tertiary-disabled-border-style
-  $sprk-btn-tertiary-disabled-border-color !default;
-/// The border of the Quaternary Button component when disabled on hover.
-$sprk-btn-quaternary-disabled-hover-border: $sprk-btn-quaternary-disabled-border-width
-  $sprk-btn-quaternary-disabled-border-style
-  $sprk-btn-quaternary-disabled-border-color !default;
-
-// Secondary
 /// The background color of the Secondary Button component.
 $sprk-btn-secondary-background-color: transparent !default;
 /// The border color of the Secondary Button component.
 $sprk-btn-secondary-border-color: $sprk-red !default;
 /// The border of the Secondary Button component.
 $sprk-btn-secondary-border: $sprk-btn-border-width $sprk-btn-border-style
-  $sprk-btn-secondary-border-color !default;
```

Comment:
Double check this value to make sure it matches the old one. Check the old setting in main branch and compare it to what is set in button.json token file.